applications/MACI-CDao.md

```diff
@@ -0,0 +1,268 @@
+
+- How did you identify these needs? Please provide evidence in the form of (scientific) articles, forum discussions, case studies, or raw data.
+
+  The implementation of anti-collusion is based on super-fair governance. The concept of super fair governance comes from super fair distribution. Super fair distribution is a mathematical model for solving cake-cutting problems.Typical cake-cutting solutions are envy-free division and super-fair distribution, two decentralized decision-making techniques based on unanimous consent. These techniques aim to make everyone believe that their cake is bigger than that of others or that their cake is bigger than that of the average.
+
+  Related papers can be referred to: 
+  1 Arnsperger C. Envy-Freeness and Distributive Justice[J]. Journal of Economic Surveys , 1994 , 8(2):155-186. 
+  2 Brams S J, Taylor AD. Fair Division: From Cake-Cutting to Dispute Resolution[J] Social Justice Research,1999,12(2). 
+  3 Liu Chao, Wang WenJie,Super Fair Distribution Based on Individual Preference[C],International Conference on Management Science And Intelligent Control(ICMSIC 2011), 2011-8:368-372. 
+  4 Chen Y , Lai J K , Parkes D C ,et al. Truth, justice, and cake cutting[J].Games & Economic Behavior, 2013, 77(1):284-297. 
+
+  The papers mentioned above are aimed at divisible resources, but by introducing economic compensation measures, they can be applied to indivisible resources. More importantly, we can extract several key points of decentralized decision-making from it:\\
+  1 All participants have decision-making power;\\
+  2 The final decision reflects the preferences of all participants.\\
+  Obviously, the second feature has already achieved the anti-collusion mechanism we expected to achieve.
+
+
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+
+  There is no such pallet that supports Minimal anti-collusion infrastructure. The reasons for the failure of these projects are as follows:  \\
+  The existing voting system, especially based on the principle of majority, will inevitably damage the rights of minority opinion holders and cannot avoid collusion among voters to obtain improper benefits. \\
+  Encryption methods to assist in voting decisions can only ensure that participants accurately do what they want but cannot guarantee that the system does the right thing.  \\  
+  The solution to the problem lies in changing the principle of majority voting to the principle of unanimous consent. In other words, all voters must unanimously agree upon any final decision.
+ 
+- Are there any projects similar to yours in related ecosystems? 
+
+  There is no such projects in related ecosystems.
+
+
+## Team : 
+
+### Team members
+
+- Name of team leader:    LIU Chao
+- Names of team members:  Sam
+
+### Contact
+
+- **Contact Name:**       LIU Chao
+- **Contact Email:**      liu_chao@aliyun.com 
+
+### Legal Structure
+
+- **Registered Address:**       NA
```

Comment:
If you have a legal entity then please also list the address here. Otherwise you can apply as an individual.

Reply:
Thank you for your correction. I will address this issue later.